package.json

```diff
@@ -20,7 +20,7 @@
   ],
   "scripts": {
     "build": "npm run compile",
-    "clean": "rm -rf ./build/",
+    "clean": "shx rm -rf ./build/",
```

Comment:
Thank you for the contribution!

`shx` seems like a good utility but we already have `rimraf` installed for doing `rm -rf`. Can you please just use it instead of installing a new package? Thanks!

Reply:
ok.

well, should I rebase? ( https://github.com/google/ts-style/pull/126/commits/ff140de4fc550ac88daa1cfb5b5c5bc398ea0c85 commit goese away )